Source/Mosa.UnitTests/ComplexOptimizationTests.cs

```diff
@@ -252,5 +252,17 @@ public static bool OptimizationTest35(int a, int b)
 			int q = z & 0xF;
 			return p >= 16 && q == 0;
 		}
+
+		//[MosaUnitTest(1)] // inifinit loop --- so can not uncomment the test
```

Comment:
Could set the value to 1000, then it wouldn't cause an infinite loop

Reply:
This is by design - infinity loop.

I used this to check an optimization.